### DISCUSSION TITLE
Visual Scene Display hotspot border

### DISCUSSION THREAD

**Discussion by aetkie** [UNC-Project-Open-AAC/OS-DPI#155] [Ideas]
@lageist  and I are wondering, is there a way to make the hotspot borders invisible or even less visible with thinner borders?

**Comment by gbishop** [comment]
If you have a field in your spreadsheet named "invisible" with a TRUE
value, it should not show up. Is that what you want?

**Comment by aetkie** [comment]
Yes, that's it! Thank you.
